setup.cfg

```diff
@@ -24,22 +24,19 @@ classifiers =
 [options]
 zip_safe = true
 include_package_data = true
-python_requires = >= 2.7, != 3.0.*, != 3.1.*, != 3.2.*
-#, != 3.3.*
+python_requires = >= 2.7, != 3.0.*, != 3.1.*, != 3.2.*, != 3.3.*
 setup_requires = 
     setuptools
 install_requires = 
     setuptools
 
 #test_suite = tests.tests.Tests
-dependency_links = ['git+https://github.com/devttys0/yaffshiv.git']
-#'git+https://github.com/devttys0/sasquatch.git',
 
 [options.extras_require]
 plots = matplotlib
 x86 = capstone
-YAFFS = yaffshiv
-#SQUASH = sasquatch
+#YAFFS = yaffshiv @ git+https://github.com/devttys0/yaffshiv.git
```

Comment:
This breaks things. You need to comment out the part starting from `@`, not to comment out the line entirely.